File: 11_orden_por_filas_2d.cs
int[,] mat = {{3,1,2},{9,5,6}};
for(int i=0; i<mat.GetLength(0); i++){
    int[] fila = new int[mat.GetLength(1)];
    for(int j=0;j<fila.Length;j++) fila[j]=mat[i,j];
    Array.Sort(fila);
    for(int j=0;j<fila.Length;j++) mat[i,j]=fila[j];
}

File: 11_orden_por_filas_2d.java
import java.util.*;
public class Main {
    public static void main(String[] args){
        int[][] mat = {{3,1,2},{9,5,6}};
        for(int[] fila: mat) Arrays.sort(fila);
        for(int[] f: mat) System.out.println(Arrays.toString(f));
    }
}